src/auth/auth.service.ts

```diff
@@ -155,7 +141,9 @@ export class AuthService {
   }
 
   async getTokensByUsername(userName: string): Promise<AuthToken[]> {
```

Comment:
I think we should change this also to `user: User`.

Reply:
As I didn't change this code in this PR, I'd suggest to open an extra PR for that…